environment.yml

```diff
@@ -4,7 +4,7 @@ channels:
   - bioconda
   - conda-forge
 dependencies:
-  - anaconda::openjdk=8.0.152
+  - conda-forge::openjdk=8.0.144
```

Comment:
Just curious about usage of various java versions in BioInformatics, especially `jdk-8`.

I have tested `nextflow` with `jdk-11` and things do seem to work fine. Despite the major changes from `jdk-9` onwards, please correct me if I'm wrong, I think most java dependent tools ought to work fine even with the latest `LTS jdk-11` right? 🤔

Reply:
I'm not sure why most tools 'require' 8. Maybe @apeltzer knows why as the resident Java programmer ;).

We are here just using what the tools require in the conda recipes.

In this case more specifically - the issue is the malt dependency in the HOPS package apparently _requires_ specifically 8.0.144 for some reason.

This was leading to conda build failures (also why I included the new GitHub action run)

 The anaconda channel doesn't have  this version, thus the switch. This version is also valid for the dependencies so I thought it's fine for now (I'm speaking to a colleague whether he knows if the dependency can be made looser).

Reply:
Ahh, I see. Thanks for the wonderful explanation @jfy133 - appreciate it😊